docs/ci-cd-environment/install-self-hosted-agent.md

```diff
@@ -4,53 +4,62 @@ description: This guide describes how to install a self-hosted agent on various
 
 # Installing a self-hosted agent
 
-The agent is open source and can be found at [https://github.com/semaphoreci/agent](https://github.com/semaphoreci/agent).
+The agent is open source and can be found at [https://github.com/semaphoreci/agent](https://github.com/semaphoreci/agent). The host requirements are:
+
+- bash
+- git
+- docker - [manageable without sudo][docker without sudo]
+- docker-compose
 
 ## Installing the agent on Ubuntu/Debian
 
-Prepare your machine:
+<b>1. Prepare your machine:</b>
 
 ```
 sudo mkdir -p /opt/semaphore/agent
 sudo chown $USER:$USER /opt/semaphore/agent/
 cd /opt/semaphore/agent
 ```
 
-Download the agent:
+<b>2. Download the agent:</b>
 
 ```
 curl -L https://github.com/semaphoreci/agent/releases/download/v2.0.11/agent_Linux_x86_64.tar.gz -o agent.tar.gz
 tar -xf agent.tar.gz
 ```
 
-Install the agent:
+<b>3. Install the agent:</b>
 
 ```
 sudo ./install.sh
 ```
 
-The script asks for your Semaphore organization, the agent type registration token and the Linux user to use to run the service. After that, it creates and starts the systemd service.
+The script asks for your Semaphore organization, the agent type registration token and the Linux user to use to run the service and does the following:
```

Comment:
I find this part confusing `the agent type registration token`. What does it mean? Can it be simplified?

I think this `the Linux user to use to run` can be changed to `the Linux user used to run`.

Reply:
There's another part of the documentation that explains what that token is. I think we can add a link to that part here to make things clearer.